deployment/docker-compose.yml

```diff
@@ -8,17 +8,6 @@
 
 # Our main nginx service - this is for testing only - normally
 # You would use a standalone nginx instance running on the host!
-db:
-  # Note you cannot scale if you use conteiner_name
-  container_name: channel-api-db
-  image: kartoza/postgis:9.4-2.1
-  volumes:
-    - ./pg/postgres_data:/var/lib/postgresql
-    - ./backups:/backups
-  environment:
-    - USERNAME=docker
-    - PASS=docker
-  restart: always
 
 uwsgi-base:
```

Comment:
Why is this called uwsgi? Why do we need a base image?

Reply:
- uwsgi is a communication protocal implementation of the uWSGI server. 
- Hence it was used because we were using the wsgi and not daphne server :-) , So umm we do not need it anymore as we are no longer using wsgi.
- A base image is what the other image (s) is built on top of.
If I start with a uwsgi-base image then add web on top to create a webserver image then the uwsgi-base image is the base image.
Am updating my PR :-).